src/Form/Type/DataFlowType.php

```diff
@@ -17,9 +17,11 @@
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\Extension\Core\Type\CollectionType;
 use Symfony\Component\Form\Extension\Core\Type\FormType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
```

Comment:
Is this `use` actually used?

Reply:
You're right, must have sneaked in while I wasn't looking